Simplenote/Note+Interlink.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+
+// MARK: - Note + Interlink
+//
+extension Note {
+
+    /// Returns the receiver's Markdown Internal Reference, when possible
+    ///
+    var markdownInternalLink: String? {
+        guard let title = titlePreview, let key = simperiumKey else {
+            return nil
+        }
+
+        let shortened = title
```

Comment:
It looks like this approach can truncate a title in the middle of a word. Maybe not a big deal unless you have a very long title, but would it make sense to check for the nearest word break to the target length and truncate from there?  It would avoid chopping a word in the middle. Maybe the fixed length as a fallback when there are no spaces within a reasonable distance? 
Just a thought.